### DISCUSSION TITLE
Is it possible to cancel caching of the final result to to generate with the same parameters?

### DISCUSSION THREAD

**Discussion by LEv145** [comfyanonymous/ComfyUI#211] [Q&A] [answered]
When generating images with exactly the same input data, there is still a side effect that makes them different:
- strange scarf and strange hair:
![ComfyUI_00046_](https://user-images.githubusercontent.com/40071054/226757497-aef565fc-6b42-4892-b902-af4cd64753b8.png)
- strange hair:
![ComfyUI_00044_](https://user-images.githubusercontent.com/40071054/226757507-ad47d19f-0f49-41bc-9940-4eb16e2473f8.png)
- not bad:
![ComfyUI_00042_](https://user-images.githubusercontent.com/40071054/226757514-6084aad6-1092-4f9e-8346-318edeaa6667.png)

I would like to use this as sometimes it can solve some picture problems, but because of caching, you have to change prompt, and then return
This function works in other UIs

Template: https://comfyanonymous.github.io/ComfyUI_examples/area_composition/square_area_for_2_subjects.png

**Comment by comfyanonymous** [comment, marked as answer]
if you add one or more spaces to your text prompt it won't actually change the tokens but it will rerun it.

The side effect is probably due to the SDE sampler and something I will need to fix.

**Reply by LEv145** [reply]
Thank you, hope for improvement, good luck!

But such the side effect is sometimes convenient, as I showed above, I would like to use it or something similar

**Reply by LEv145** [reply]
@comfyanonymous This behavior, as I understand it, is due to `xformers`

**Reply by comfyanonymous** [reply]
starting from 0.0.19 xformers should be deterministic.

**Reply by fractal-fumbler** [reply]
regarding SDE and it's deterministic abilities 
getting this warning
```python
/tmp/ComfyUI/auto/lib/python3.10/site-packages/torchsde/_brownian/brownian_interval.py:594: UserWarning: Should have tb<
=t1 but got tb=5.516684055328369 and t1=5.516684.
  warnings.warn(f"Should have {tb_name}<=t1 but got {tb_name}={tb} and t1={self._end}.")
```

maybe related
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/5210
https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/7730
hope it helps
